About.html

```diff
@@ -0,0 +1,116 @@
+            </ul>
+        </nav>
+      </div>
+    </header>
+  <main class="cultural-days-wrap" id="main">
+    <section class="about new">
+      <h3 class="about-tite2">"hello! sharing world </h3>
+      <h1 class="about-titel">WORLDWIDE EVENT PROMOTER</h1>
+      <div class="invite-wrap invite-wrap-about">
+        <p class="invite-msg global">International Conference on Climate Change aims to bring together  scientists, researchers and research scholars to exchange and share their experiences and research results.</p>
+      </div>
+      <p class="text-about">Please contact us per Email for any further questions about CC Global Summit 2015!</p>
+      <p class="locate">Conference@newa123</p> 
+    </section>
+    <section class="logo" id="logo" >
+      <h2 class="secondary-tittle logo-tittle">WORLD CONFERENCE ALERTS 2022, 2023 Logo</h2>
+      <div class="bar-icon"></div>
+      <p class="invite-msg logo-msg">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minima explicabo inventore omnis necessitatibus molestias labore rerum beatae consequuntur, et eius vitae recusandae, quis quidem. Minus fugit excepturi aut nemo mollitia.</p>
+      <img class="logo-img" src="images/logo.png" alt="logo " width="572" height="206">
+    </section>
+    <a href="#locate"></a><div  class="separeter"></div>
+    
+    <section class="logo" id="gallery">
+      <h2 class="secondary-tittle logo-tittle">See the past CC Global summits</h2>
+      <div class="bar-icon"></div>
+      <p class="invite-msg logo-msg">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minima explicabo inventore omnis.</p>
+      <div class="img-past-wrap">
+        <img class="logo-img-past" src="images/about_2013.png" alt="CC Global summit  2013" width="455" height="252">
+        <img class="logo-img-past" src="images/about_2011.png" alt="CC Global summit  2011" width="455" height="252">
+      </div>
+    </section>
+    <section class="featured-activities" id="off"></section>
+    <section class="partners" id="sponsor2">
+      <h2 class="secondary-tittle partner-tittle">Partner</h2>
+      <div class="bar-icon"></div>
+      <div class="partners-wrap">
+        <ul class="partners-list">
+          <li class="partner ">
+            <a href="#"><img src="images/mozilla.png" alt="mozzila logo" width="150" height="50"></a>
+          </li>
+          <li class="partner1 ">
+            <a href="#"><img src="images/googlekorea.png" alt="google logo" width="150" height="50"></a>
+          </li>
+          <li class="partner1 ">
+            <a href="#"><img src="images/naver.png" alt="naver logo" width="130" height="50"></a>
+          </li>
+          <li class="partner1 ">
+            <a href="#"><img src="images/kakao.png" alt="Kakao logo" width="120" height="50"></a>
+          </li>
+          <li class="partner1 ">
+            <a href="#"><img src="images/airbnb.png" alt="Airbnb logo" width="130" height="50"></a>
```

Comment:
- [ ] [Optional]  Changing the CSS property from `space-around` to `space-evenly` on the partners section improves the design.